scripts/visualize/visualizeWalleyeDecline.R

```diff
@@ -20,7 +20,7 @@ svgFishTrends <- function(object, filename){
   dinosvg:::write_svg(svg, file = filename)
 }
 
-visualize.visualizeFishTrends <- function(item){
+visualize.visualizeFishTrends <- function(outfile, ...){
   fish <- readData(item$filename)
```

Comment:
maybe - it gets ignored for now. i would like to implement args soonish, so you could keep it in as a placeholder if you want...